frontend/DataView/DataView.js

```diff
@@ -74,9 +74,20 @@ class DataView extends React.Component<DataViewProps> {
       return <div style={missingStyle(theme)}>null</div>;
     }
 
+    const dataType = typeof data;
+    var isSingleValue = dataType === 'number' || dataType === 'string' || dataType === 'boolean';
     var isArray = Array.isArray(data);
     var elements = [];
-    if (isArray) {
+    if (isSingleValue) {
+      elements.push(<DataItem
+          name={''}
```

Comment:
Even though this is a single value, since we're adding it to an array– it needs a `key`.